cfgov/v1/models/atoms.py

```diff
@@ -53,8 +53,13 @@ class Hyperlink(blocks.StructBlock):
     text = blocks.CharBlock(required=False)
     url = blocks.CharBlock(default='/', required=False)
 
-    def __init__(self, required=True):
+    def __init__(self, required=True, text=None, url=None):
         self.required = required
+        if text:
+            self.base_blocks['text'].meta.default = text
+        if url:
+            self.base_blocks['url'].meta.default = url
+
```

Comment:
hmm i'm seeing the case where if you set that text param somewhere. it'll be set for ALL hyperlinks everywhere instead of that one instance